confluent_kafka/src/confluent_kafka.c

```diff
@@ -249,6 +249,39 @@ static PyObject* KafkaError_richcompare (KafkaError *self, PyObject *o2,
 	return result;
 }
 
+static PyObject *KafkaError_new (PyTypeObject *type, PyObject *args,
+                                 PyObject *kwargs) {
+        return type->tp_alloc(type, 0);
+}
+
+/**
+ * @brief Initialize a KafkaError object.
+ */
+static void KafkaError_init (KafkaError *self,
+			     rd_kafka_resp_err_t code, const char *str) {
+	self->code = code;
```

Comment:
Since this code is moved around we can fix the spacing, convert tabs to 8 whitespaces.